crates/abq_queue/src/queue.rs

```diff
@@ -65,7 +65,13 @@ enum RunState {
         /// manifest being generated.
         worker_connection_times: Mutex<WorkerSet<Instant>>,
 
+        /// The number of tests to batch to a worker at a time, as hinted by an invoker of the work.
+        // needed to pass through to HasWork
```

Comment:
I would avoid this comment because it can get out-of-date. Find-references in an editor should show you where it's used.